src/basic/fd-util.h

```diff
@@ -52,6 +52,11 @@ static inline void fclosep(FILE **f) {
         safe_fclose(*f);
 }
 
+static inline void *close_fd_ptr(void *p) {
```

Comment:
```suggestion
static inline void* close_fd_ptr(void *p) {
```

(Not your fault. But if you touch this, please write T* for function return types.

Reply:
Done